packages/core/src/odata/v4/index.ts

```diff
@@ -15,4 +15,5 @@ export * from './extract-odata-etag';
 export * from '../common';
 export * from '../common/selectable/one-to-many-link';
 
+export { filterFunction } from './filter-function';
```

Comment:
Is this only exported for type testing?

Reply:
No, it is exported for usage. v2 uses the export from common, because it doesn't change it. As I had to duplicate the boilerplate anyways, I can actually move it to v2 as well.

Reply:
Ah no, I can't because it is used in common, for the common filter functions.